src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -264,6 +276,42 @@ class EthService(
     }
   }
 
+  def getCode(req: GetCodeRequest): ServiceResponse[GetCodeResponse] = {
+    Future.successful {
+      resolveBlock(req.block).map { block =>
+        val world = InMemoryWorldStateProxy(blockchainStorages, Some(block.header.stateRoot))
```

Comment:
@adamsmo I've added the tests for `getUncleCountByBlockNumber`, `getUncleCountByBlockHash` and `getBlockTransactionCountByNumber`. I'm still struggling to provide tests data for `getCode`.